src/ui/scrubber.ts

```diff
@@ -0,0 +1,131 @@
+import { Container, Label, SelectInput, SliderInput } from 'pcui';
+
+import { Events } from '../events';
+import { Tooltips } from './tooltips';
+
+class Scrubber extends Container {
```

Comment:
I'd probably called this `Timeline` but hey.

Reply:
I was being a little lazy, but I've renamed thanks!